addons/mail/models/mail_mail.py

```diff
@@ -513,6 +542,11 @@ def send(self, auto_commit=False, raise_exception=False):
 
     def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
         IrMailServer = self.env['ir.mail_server']
+        # Only retrieve recipient followers of the mails if needed
+        recipients_follower_status = (
+            self.env['mail.followers']._get_mail_recipients_follower_status(self.ids) if self.filtered(
```

Comment:
Technically, if only 1 mail of the batch has 'mail/unfollow' in body, we are going to compute the follwoer status for all of them ... shouldn't we filter and then call the method on the filtered set ? Anyway the key (model, res_id) is used afterwards so it would be safe.

Reply:
Indeed, that’s good idea.